package com.sinalez.sinaleasy_back.exceptions.customExceptions.signalException;

public class SignalNotFoundException extends RuntimeException {
    public SignalNotFoundException() {
        super("Sinal n√£o encontrado!");
    }

    public SignalNotFoundException(String message) {
        super(message);
    }
    
}
